newsroom/wire/views.py

```diff
@@ -141,6 +141,7 @@ def get_home_data():
         "cards": cards,
         "products": get_products_by_company(company) if company else [],
         "user": str(user["_id"]) if user else None,
+        "userProducts": user.get("products") if user else None,
```

Comment:
this should probably also include company products without seats limit

Reply:
we already include company products here `"products": get_products_by_company(company) if company else [],`

Reply:
yep sure it's there, but it's not taken into account for that more news button, is it?

Reply:
yep we are using these products and filter that based on card config.